File: JavaBasics/Conditional Statements/src/AreaOfFigures.java
import java.util.Scanner;

public class AreaOfFigures {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);
        String figure = scanner.nextLine();
        double area = 0;

        switch (figure) {
            case "square":
                double a = Double.parseDouble(scanner.nextLine());
                area = a * a;
                break;
            case "rectangle":
                double c = Double.parseDouble(scanner.nextLine());
                double b = Double.parseDouble(scanner.nextLine());
                area = c * b;
                break;
            case "circle":
                double d = Double.parseDouble(scanner.nextLine());
                area = Math.PI * d * d;
                break;
            case "triangle":
                double e = Double.parseDouble(scanner.nextLine());
                double f = Double.parseDouble(scanner.nextLine());
                area = (e * f) / 2;
                break;

        }
        System.out.printf("%.3f", area);
    }
}

File: JS/Basics/02.Conditional Statements - Lab/areaOfFigures.js
function areaOfFigures(input){
    let area = 0.0;

    if(input[0] === "square"){
        let a = Number(input[1]);
        area = a*a;
    } 
    else if(input[0] === "rectangle"){
        let sideA = Number(input[1]);
        let sideB = Number(input[2]);
        area = sideA * sideB;
    }
    else if(input[0] === "circle"){
        let radius = Number(input[1]);
        area = Math.PI * Math.pow(radius,2);
    }
    else if(input[0] === "triangle"){
        let base = Number(input[1]);
        let height = Number(input[2]);
        area = 0.5 * base * height;
    }
    console.log(area.toFixed(3));
}
areaOfFigures(["circle","6"]);